eng/Generate.ps1

```diff
@@ -84,7 +84,7 @@ if (!($Exclude -contains "TestServer"))
     }
 }
 
-$llcArgs = "--low-level-client=true --credential-types=AzureKeyCredential --credential-header-name=Fake-Subscription-Key"
+$llcArgs = "--low-level-client=true --security=AzureKey --security-header-name=Fake-Subscription-Key"
```

Comment:
Don't we support no-cred LLC now?

Reply:
Yep, I could just drop this to be honest. I just figured the key credential was more of the "happy path" than the uncommon no-auth.